### ISSUE TITLE
Logs from an internal library are leaking into the GitHub action

### ISSUE THREAD

**Issue by rberrelleza** [okteto/deploy-preview#21]
![image](https://user-images.githubusercontent.com/475313/131381443-636c6c64-4f2c-43ec-a0fa-d24931b68cf5.png)

**Comment by rberrelleza** [comment]
This is related to https://github.com/octokit/octokit.rb/issues/1357

**Comment by pchico83** [comment]
@rberrelleza should we close the issue then?

**Comment by rberrelleza** [comment]
@pchico83 no, I think we should still validate and pin the version once the issue in octokit is fixed.
